ccip-sdk/src/evm/decode.test.ts

```diff
@@ -319,6 +319,47 @@ describe('EVMChain.decodeMessage', () => {
       assert.equal(ta.amount, 5000n)
       assert.equal(ta.destGasAmount, destGasAmount)
     })
+
+    it('should not have numeric keys in decoded message', () => {
```

Comment:
test to grab the issue I mentioned in the previous comment about the evm side request showing the numeric keys